doc/Publishing.md

```diff
@@ -106,51 +106,95 @@ Format:
 
 ### 2.1 Performing a Release
 
-- Announce that the release is starting in the discussion.
+- Announce that the release is starting in the discussion (not needed for patch releases):
 
-  > The release will start now, We’ll post an update once it has completed.
+  > The release will start now. We’ll post an update once it has completed.
+
+- Ensure the release branch is checked out (e.g., `release/1.55.x`).
+
+- Clean the working directory:
+
+  ```bash
+  git clean -xdf
+  ```
 
 - Build the changes:
 
   ```bash
   yarn
   ```
 
-- Confirm the changes are built (individual `@theia` extensions should have their `lib/` folders present).
-- Perform the release using:
+- Confirm the changes are built (ensure `@theia` extensions have their `lib/` folders).
+
+- Create a short-lived granular npm auth token ([instructions](https://docs.npmjs.com/creating-and-viewing-access-tokens#creating-granular-access-tokens-on-the-website)):
+  - Expiration: `7 days`.
+  - Under Packages and scopes:
+    - Permissions: `Read and write`.
+    - Select `Only select packages and scopes`
+    - Select packages and scopes: `@theia`.
+
+- Set the token:
+
+  ```bash
+  npm set "//registry.npmjs.org/:_authToken=${TOKEN}"
```

Comment:
I usually do this with a leading space. On many shells, this prevents the command from being added to the shell's history.

Reply:
Good point. Added a commit for this :+1: